src/main/java/de/numcodex/sq2cql/model/Mapping.java

```diff
@@ -98,4 +110,12 @@ public Map<TermCode, AttributeMapping> attributeMappings() {
     public Optional<String> timeRestrictionPath() {
         return Optional.ofNullable(timeRestrictionPath);
     }
+
+    public Optional<String> codeFhirPath() {
```

Comment:
Please explain what the `codeFhirPath` is including an example.

Reply:
codeFhirPath is the fhirPath the key is applied to. Currently, we don't have a use case for it.